src/main/java/uk/gov/companieshouse/company/profile/service/CompanyProfileService.java

```diff
@@ -184,6 +184,52 @@ public void addExemptionsLink(String contextId, String companyNumber) {
         }
     }
 
+    /**
+     * Delete an exemptions link for a company profile and call chs-kafka-api
+     * to notify a resource has been changed.
+     *
+     * @param contextId Request ID from request header "x-request-id
+     * @param companyNumber The number of the company to update
+     */
+    public void deleteExemptionsLink(String contextId, String companyNumber) {
+        try {
+            CompanyProfileDocument document = companyProfileRepository.findById(companyNumber)
+                    .orElseThrow(() -> new DocumentNotFoundException(
+                            String.format("No company profile with company number %s found",
+                                    companyNumber)));
+
+            if (StringUtils.isBlank(document.getCompanyProfile().getLinks().getExemptions())) {
+                logger.error("Exemptions link for company profile already does not exist");
```

Comment:
Could be worded better e.g.:

logger.error("Exemptions link for company profile does not exist");